docs/conf.py

```diff
@@ -61,9 +61,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '0.5.0'
+version = '0.7.7'
```

Comment:
Could we just import the version from the twitch observer package? Similar to how setup.py works?

```python
from twitchobserver import __version__
```